app/controllers/users/settings_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Users::SettingsController < ApplicationController
+  def update
+    current_user.settings(:user_settings).color_scheme
+    current_user.settings(:user_settings).update!(color_scheme: params.require(:color_scheme))
+
+    redirect_to :user_reviews
```

Comment:
My knowledge with rails is limited but why do we need to redirect to user reviews page after updating the theme settings? It should stay on the same page(`My Dashboard`), right?

Reply:
The route naming is setup a bit weirdly. `user/<user_id>/routes` actually point to the dashboard. It shouldn't but can fix that separately.